doc/user-guide/cli.md

```diff
@@ -94,11 +94,11 @@ ploomber-cloud examples flask/basic-app
 
 A full list of example applications is available [here](https://github.com/bryannho/doc/tree/main/examples)
 
-## Defining environment variables
+## Defining secrets
 
-If your project uses environment variables, you can define them in an `.env` file.
+If your project uses secrets, you can define them in an `.env` file.
 
-In your main project directory, create an `.env` file. Open it in your code editor, and enter your environment variables. It should look like this:
+In your main project directory, create an `.env` file. Open it in your code editor, and enter your secrets. It should look like this:
```

Comment:
for consistency `MY_ENV_VAR_1` and `MY_ENV_VAR_2` can be renamed to `MY_SECRET_1` and `MY_SECRET_2`

Reply:
I agree